tools/distpartitioning/utils.py

```diff
@@ -402,21 +398,31 @@ def get_idranges(names, counts):
     gnid_end = gnid_start
     tid_dict = {}
     gid_dict = {}
+    orig_num_chunks = 0
     for idx, typename in enumerate(names): 
         type_counts = counts[idx]
         tid_start = np.cumsum([0] + type_counts[:-1])
         tid_end = np.cumsum(type_counts)
         tid_ranges = list(zip(tid_start, tid_end))
 
-        type_start = tid_ranges[0][0]
-        type_end = tid_ranges[-1][1]
-
         gnid_end += tid_ranges[-1][1]
 
         tid_dict[typename] = tid_ranges
         gid_dict[typename] = np.array([gnid_start, gnid_end]).reshape([1,2])
 
         gnid_start = gnid_end
+        orig_num_chunks = len(tid_start)
+
+    assert expected_num_chunks <= orig_num_chunks, \
+        'expected num_chunks should be less/euqual than original one.'
```

Comment:
What is "original one" here? The error message will not make any sense to the users.